app/core/user.py

```diff
@@ -5,10 +5,11 @@ class User:
     def __init__(self):
         pass
 
-    def __init__(self, userId, name, password):
+    def __init__(self, userId, name, password, capstone):
```

Comment:
make capstone false by default, will not break existing functionality. e.g. ', capstone = false'